### PR TITLE
Perses finale

### PR DESCRIPTION
**Pull request by triphora**
Removal of v1, moving of v2, and maybe the start of v3?

Still very WIP and still trying to work out what the scope of this PR is. Making this PR because I much prefer the GitHub diff view over using plain `git diff`.

Closes MOD-217
Closes MOD-221
Closes MOD-251

### CODE REVIEW

File: src/routes/mod.rs [hunk 1]
```diff
@@ -1,200 +1,35 @@
-mod v1;
-pub use v1::v1_config;
+use crate::file_hosting::FileHostingError;
+use actix_web::http::{Method, StatusCode};
```
Comment (Geometrically) [inline src/routes/mod.rs:2]: unused import `Method`

File: src/routes/mod.rs [hunk 2]
```diff
@@ -1,200 +1,35 @@
...
+use actix_web::{web, HttpResponse};
+use chrono::{Timelike, Utc};
```
Comment (Geometrically) [inline src/routes/mod.rs:4]: unused imports

**Review by Geometrically — CHANGES_REQUESTED** [review]
looks good besides some minor things. have you tested that the new route registration works?

**Comment by triphora** [comment]
As far as I can tell it works

**Review by Geometrically — APPROVED** [review]

**Pull request merged**